numba_dppy/tests/__init__.py

```diff
@@ -14,3 +14,4 @@
 
 import numba_dppy
 from numba_dppy import config
+from numba_dppy.tests.njit_tests.dpnp import *
```

Comment:
Why is this?

Reply:
There is a dpnp skip condition:  **pytestmark = pytest.mark.xfail(not ensure_dpnp(), reason="No dpnp available")** In **numba_dppy.tests.njit_tests.dpnp.__init__**  . I import it to been available in this module too

Reply:
It is not clean and obvious.